saleor/graphql/order/types.py

```diff
@@ -1089,3 +1096,13 @@ def wrap_voucher_with_channel_context(data):
     @traced_resolver
     def resolve_language_code_enum(root, _info, **_kwargs):
         return LanguageCodeEnum[str_to_enum(root.language_code)]
+
+    @staticmethod
+    @traced_resolver
+    def resolve_errors(root, _info, **_kwargs):
+        country = get_order_country(root)
+        try:
+            validate_draft_order(root, country)
```

Comment:
This function could generate a lot of DB queries.

Reply:
Will be optimized in separate PR.

Reply:
https://app.clickup.com/t/2549495/SALEOR-3355